pkg/controller/buildrun/runtime_image_test.go

```diff
@@ -0,0 +1,88 @@
+package buildrun
+
+import (
+	"fmt"
+
+	buildv1alpha1 "github.com/redhat-developer/build/pkg/apis/build/v1alpha1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("runtime-image", func() {
```

Comment:
I think we should also add e2e test to cover this new feature.

Reply:
Sure, I'll do in the near future.

Reply:
Please consider the latest commits.